### PR TITLE
upgrade to kotlin 1.6.10

### PR DESCRIPTION
**Pull request by AndreasMattsson**
Upgrade to kotlin 1.6.10 and kotlinxCoroutines 1.5.2

### CODE REVIEW

**Comment by JakeWharton** [comment]
This will require Compose 1.1.0 which is not yet released.

**Comment by AndreasMattsson** [comment]
Alright, I get it if you don't want to include code dependent on mere release candidates.

**Comment by JakeWharton** [comment]
Thankfully we have only but a week to wait.

File: dependencies.gradle [hunk 1]
```diff
@@ -1,7 +1,7 @@
 buildscript {
 	ext.versions = [
-		'kotlin': '1.5.30',
-		'kotlinxCoroutines': '1.5.0',
+		'kotlin': '1.6.10',
+		'kotlinxCoroutines': '1.5.2',
```
Comment (opatry) [inline dependencies.gradle:4]: Shouldn't be 1.6.0?

**Comment by AlexBeggs** [comment]
Can this be bumped now?

**Comment by JakeWharton** [comment]
Yes!

**Comment by saket** [comment]
~~@AndreasMattsson wanna try going to Kotlin 1.7.10?~~

Update: my attempt: https://github.com/JakeWharton/mosaic/pull/54

**Comment by JakeWharton** [comment]
Obsoleted by #54, thanks.

**Pull request closed without merging**